### ISSUE TITLE
Type pages 21-30

### ISSUE THREAD

**Issue by PashPaw** [sebras/JOSS-I#8]

**Comment by PashPaw** [comment]
[JOSS-I-21.txt](https://github.com/sebras/JOSS-I/files/6073746/JOSS-I-21.txt)
[JOSS-I-22.txt](https://github.com/sebras/JOSS-I/files/6073747/JOSS-I-22.txt)
[joss-i-23.txt](https://github.com/sebras/JOSS-I/files/6073748/joss-i-23.txt)

**Comment by sebras** [comment]
Thank you! I'll stick them on master immediately!

I planned to do this a long time ago, but then something happened and I forgot. I'm sorry about that, but I appreciate your effort! :)

**Comment by sebras** [comment]
I decided to put them on [a separate branch](https://github.com/sebras/JOSS-I/commits/sebras/pages/21-30). That way I can rebase things if needed without anyone complaining. I fixed a few small indentation things so your transcribed pages line up with the previous ones.

I transcribed page 30. Should you decide to (but you don't have to) transcribe page 24 just attach it here like you did and I'll make sure it won't be lost. :) I'm on to tackling page 29 now!
